src/cron/schedule-code-review.js

```diff
@@ -92,36 +106,27 @@ const checkThreadsForReview = async (client, statusText) => {
 };
 
 /**
- * Schedules a cron job to run `checkThreadsForReview` at specified intervals.
- *
- * @param {Client} client - The Discord.js client instance.
+ * Schedules cron jobs for all statuses defined in the configuration.
+ * @param {Client} client - Instance of the Discord.js client.
  */
-const scheduleReviewCheck = (client) => {
-  const statusText = getMappedStatusText(STATUS_KEY);
-  if (!statusText) {
-    return;
-  }
+const scheduleAllStatusChecks = (client) => {
+  Object.entries(CRON_REVIEW_SETTINGS.statusScheduleRemember).forEach(
+    ([key, config]) => {
+      const schedule = config.scheduleConfig;
 
-  const schedule = CRON_SCHEDULE_REVIEW.scheduleReview;
-
-  if (typeof schedule !== 'string' || !schedule.trim()) {
-    console.error('Invalid or missing cron schedule in configuration.');
-    return;
-  }
-
-  try {
-    new CronJob(
-      schedule,
-      () => {
-        checkThreadsForReview(client, statusText);
-      },
-      null,
-      true,
-      CRON_SCHEDULE_REVIEW.timeZone
-    );
-  } catch (error) {
-    console.error('Failed to create CronJob:', error.message);
-  }
+      try {
+        new CronJob(
+          schedule,
+          () => checkThreadsForStatus(client, key),
```

Comment:
pass the config here to the checkThreadsForStatus function

```suggestion
          () => checkThreadsForStatus(client, key, config),
```